vue-frontend/src/components/home-new/BlogSection.vue

```diff
@@ -88,12 +88,13 @@
               >
                 <div>
                   <span
+                    @click="openBlog(blog)"
```

Comment:
On mobile, we should disable it, as we are opening image also on click on it in mobile